src/i18n/fr/nav.ts

```diff
@@ -40,6 +40,7 @@ export default NavDictionary({
 	'guides/styling': 'Styles & CSS',
 	'guides/images': 'Images',
 	'guides/fonts': 'Polices de caractères',
+	'guides/media': 'Connecter un média hébergé ou un DAM',
```

Comment:
Don't hesitate if you have a better translation. The other menu entries use one line while with this translation the menu entry uses two lines. I don't know if that's acceptable.